whois-api/src/main/java/net/ripe/db/whois/api/healthcheck/HealthCheckJmx.java

```diff
@@ -0,0 +1,39 @@
+package net.ripe.db.whois.api.healthcheck;
+
+import net.ripe.db.whois.common.jmx.JmxBase;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jmx.export.annotation.ManagedOperation;
+import org.springframework.jmx.export.annotation.ManagedResource;
+import org.springframework.stereotype.Component;
+
+@Component
+@ManagedResource(objectName = JmxBase.OBJECT_NAME_BASE + "HealthCheck", description = "JMX health check operations")
+public class HealthCheckJmx extends JmxBase implements HealthCheck {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(HealthCheckJmx.class);
+
+    private boolean enabled = true;
+
+    @Autowired
+    public HealthCheckJmx() {
+        super(LOGGER);
+    }
+
+    @Override
+    @ManagedOperation(description = "Get JMX healthcheck status")
+    public boolean check() {
+        return enabled;
+    }
```

Comment:
Minor: feel free to ignore. Can we rename to isEnabled? It is not checking just enabling or disabling the health check via jmx?  I do not think it needs to implement healthcheck interface

Reply:
This method is used both by the HealthCheckService to check the status of the JMX healthcheck, and also as a JMX operation for a user to check the status. It's not enabling or disabling (those are separate operations)

Reply:
It's checking the current status